diff --git a/Makefile b/Makefile
index c6e81d52..659fa555 100644
--- a/Makefile
+++ b/Makefile
@@ -9,15 +9,6 @@ SHA=''
 arch = $(shell uname -m)
 # Run `DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect quay.io/prometheus/busybox:latest` to get SHA
 # Update at 2020.07.06
-ifeq ($(arch), x86_64)
-    # amd64
-    SHA="248b7ec76e03e6b4fbb796fc3cdd2f91dad45546a6d7dee61c322475e0e8a08f"
-else ifeq ($(arch), armv8)
-    # arm64
-    SHA="69508e8fdc516eacbacc0379c03c971e3043706cc8211e6bddb35d903edc3628"
-else
-    echo >&2 "only support amd64 or arm64 arch" && exit 1
-endif

 # Ensure everything works even if GOPATH is not set, which is often the case.
 # The `go env GOPATH` will work for all cases for Go 1.8+.
