diff --git a/Makefile b/Makefile
index 5fb123e7..464f656d 100644
--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@ arch = $(shell uname -m)
 ifeq ($(arch), x86_64)
     # amd64
     BASE_DOCKER_SHA="de4af55df1f648a334e16437c550a2907e0aed4f0b0edf454b0b215a9349bdbb"
-else ifeq ($(arch), armv8)
+else ifeq ($(arch), $(filter $(arch), armv8 aarch64))
     # arm64
     BASE_DOCKER_SHA="5591971699f6cf8abf6776495385e9d62751111a8cba56bf4946cf1d0de425ed"
 else
