diff --git a/Dockerfile b/Dockerfile
index 3f76cc2..d4b43b0 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,12 +1,13 @@
 ARG BASE_IMAGE_FULL
 ARG BASE_IMAGE_MINIMAL
+ARG TARGETARCH
 
 # Build node feature discovery
 FROM golang:1.16.7-buster as builder
 
 # Download the grpc_health_probe bin
 RUN GRPC_HEALTH_PROBE_VERSION=v0.3.1 && \
-    wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 && \
+    wget -qO/bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-${TARGETARCH} && \
     chmod +x /bin/grpc_health_probe
 
 # Get (cache) deps in a separate layer
